examples/angular/angular-metadata.tsconfig.json

```diff
@@ -11,7 +11,7 @@
         ],
         "experimentalDecorators": true,
         "types": [],
-        "module": "amd",
+        "module": "commonjs",
```

Comment:
When running under nodejs, the server module throws `define is not defined` since all the angular ngmodule files are wrapped in amd `define` functions.
Maybe we delay this feature till we're running Ivy?